src/main/java/de/tum/cit/aet/artemis/buildagent/service/SharedQueueProcessingService.java

```diff
@@ -114,9 +114,12 @@ public class SharedQueueProcessingService {
      */
     private final AtomicBoolean processResults = new AtomicBoolean(true);
 
-    @Value("${artemis.continuous-integration.pause-grace-period-seconds:15}")
+    @Value("${artemis.continuous-integration.pause-grace-period-seconds:60}")
```

Comment:
Is there a particular reason for the definition of the grace-period? Why is it increased?